angular.json

```diff
@@ -16,15 +16,8 @@
             "main": "src/main.ts",
             "tsConfig": "src/tsconfig.app.json",
             "polyfills": "src/polyfills.ts",
-            "assets": [
-              "src/assets",
-              "src/favicon.ico"
-            ],
-            "styles": [
-              "node_modules/swagger-ui/dist/swagger-ui.css",
-              "src/styles.scss",
-              "src/sass/theme.scss"
-            ],
+            "assets": ["src/assets", "src/favicon.ico"],
+            "styles": ["node_modules/swagger-ui/dist/swagger-ui.css", "src/styles.scss", "src/sass/theme.scss"],
```

Comment:
still need to declare "src/sass/theme.scss" ? since you import theme.scss , it will include in styles.scss